app/webpacker/css/darkswarm/split-checkout.scss

```diff
@@ -381,3 +381,15 @@
       }
     }  
 }
+
+// For small screen or for iOS devices, increase to 16px the input font-size
+@media screen and (max-width: 700px) {
+  .checkout-input input {
+    font-size: 16px;
+  }
+}
+@supports(-webkit-overflow-scrolling: touch) {
+  .checkout-input input {
+    font-size: 16px;
+  }
+}
```

Comment:
Why don't we do this for all devices?

Reply:
Because that (strange) zoom when focusing on inputs is an iOS thing. 
See https://stackoverflow.com/questions/2989263/disable-auto-zoom-in-input-text-tag-safari-on-iphone

I thought It was also a good idea to increase for small width devices (`max-width: 700px`)

Reply:
I know it's an iOS thing but why maintain two different designs? We want it to look good on iOS so if we still achieve that with this change then it will also look good on all other devices?

Reply:
I didn't want to apply this modifications to all devices with width greater than `700px` because I didn't want to change the initial design. 
Actually, what it looks like with `font-size: 16px;` on `input`: 

<img width="1180" alt="Capture d’écran 2023-01-18 à 09 29 11" src="https://user-images.githubusercontent.com/296452/213121404-791f76aa-4998-4fe4-8543-6cbd63993b76.png">

I personally don't like it, and I think it's a bit far from the original design.

Maintaining two (or more) different designs is actually a very common thing when we do responsive design. 

_... or maybe I missed your point!_

Reply:
Okay, fair enough. I personally would try to find a design that works well for all sizes but maybe that's naive. If several designs are necessary so that it always looks good then so be it.